### DISCUSSION TITLE
비선점 관련 질문

### DISCUSSION THREAD

**Discussion by IMHOSUNG** [kim6394/tech-interview-for-developer#133] [Q&A] [answered]
https://gyoogle.dev/blog/computer-science/operating-system/CPU%20Scheduling.html
해당 문서를 보다가 궁금한 점이 있어서 문의 드립니다.

비선점의 경우 처리 시간의 예측이 어렵다는 게 모호한 표현인 것 같습니다.
개인적인 생각으로는 응답시간 예측 용이 표현으로 변경이 맞는 것 같은데

제가 개념을 잘 못 이해하고 있는 건지 궁금합니다.

**Comment by gyoogle** [comment, marked as answer]
안녕하세요. 확인 결과, 해당 내용은 선점 스케줄링에 관한 내용으로 잘못 표기된 것이 맞습니다.
수정해서 다시 작성하도록 하겠습니다. 감사합니다:)
